core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -363,13 +366,22 @@ public synchronized InetSocketAddress getConfAddress() throws UnavailableExcepti
         MetricsSystem.counter(getQualifiedRetryMetricName(rpcName)).inc();
         return null;
       });
-      logger.debug("Exit (OK): {}({}) in {} ms",
-          rpcName, debugDesc, System.currentTimeMillis() - startMs);
+      long duration = System.currentTimeMillis() - startMs;
+      logger.debug("Exit (OK): {}({}) in {} ms", rpcName, debugDesc, duration);
+      if (duration >= mRpcThreshold) {
+        logger.warn("{}({}) returned {} in {} ms",
```

Comment:
Do we want to prevent double logging when log level = debug?

Reply:
this way it is easier to grep the info. I'm less concerned about the information redundancy for now